src/Altinn.Platform/Altinn.Platform.Storage/Storage/Controllers/ApplicationsController.cs

```diff
@@ -0,0 +1,326 @@
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Net;
+using System.Threading.Tasks;
+using Altinn.Platform.Storage.Models;
+using Altinn.Platform.Storage.Repository;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Documents;
+using Serilog;
+using Serilog.Core;
+
+namespace Altinn.Platform.Storage.Controllers
+{
+    /// <summary>
+    /// a summary is needed here
+    /// </summary>
+    [Route("storage/api/v1/applications")]
+    public class ApplicationsController : Controller
+    {
+        private readonly IApplicationRepository repository;
+        private Logger logger = new LoggerConfiguration()
```

Comment:
why are we creating the logger in another way then how we have done in altinn studio?

Reply:
Have redone it with dependency injection and updated docs.